Framework/Kernel/src/InternetHelper.cpp

```diff
@@ -349,7 +355,7 @@ int InternetHelper::processErrorStates(const Poco::Net::HTTPResponse &res,
   } else if ((retStatus == HTTP_FORBIDDEN) && (rateLimitRemaining == 0)) {
     throw Exception::InternetError(
         "The Github API rate limit has been reached, try again after " +
-            rateLimitReset.toSimpleString(),
+            rateLimitReset.toSimpleString() + " GMT",
         retStatus);
```

Comment:
It is worth leaving this one in place as there may well still be some places that use this class for github api communication.